rpc/README.md

```diff
@@ -6500,6 +6551,31 @@ Different reject types:
 *   `RBFRejected`: RBF rejected
 
 
+### Type `PoolTxDetailInfo`
+
+A Tx details info in tx-pool.
+
+#### Fields
+
+`PoolTxDetailInfo` is a JSON object with the following fields.
+
+*   `timestamp`: [`Uint64`](#type-uint64) - The time added into tx-pool
+
+*   `entry_status`: `string` - The detailed status in tx-pool, `Pending`, `Gap`, `Proposed`
```

Comment:
Use pending, gap, proposed to keep the capitalization consistent with other enum types.